libs/dataset-explorer/src/lib/VISIONDatasetExplorerTab.tsx

```diff
@@ -0,0 +1,331 @@
+            </Text>
+        </Stack.Item>
+        <Stack horizontal className={classNames.toolBarContainer}>
+          <Stack.Item className={classNames.searchAndFilterContainer}>
+            <SearchBox
+              className={classNames.searchBox}
+              placeholder="Search"
+            />
+            <Dropdown
+              className={classNames.filter}
+              placeholder="Filter"
+              selectedKeys={this.state.selectedFilters}
+              onChange={this.selectFilter}
+              multiSelect
+              options={FilterOptions}
+              onRenderPlaceholder={this.renderFilterIcon}
+            />
+          </Stack.Item>
+          <Slider
+            max={80} 
+            min={20}
+            className={classNames.slider} 
+            label="Thumbnail size" 
+            defaultValue={50}
+            showValue={false}
+            onChange={this.onSliderChange}
+          />
+        </Stack>
+        {this.state.selectedKey == VisionDatasetExplorerTabOptions.First && 
+        <Stack
+          horizontal={false}
+          grow
+          tokens={{ childrenGap: "l1" }}
+        >
+          <Stack horizontal className={classNames.mainContainer} style={{height: (this.state.imageDim * ROWS_PER_PAGE) + Math.floor(this.state.imageDim / 2)}}>
+            <Stack style={{ width: '50%', height: '100%' }}>
+              {/* Title Bar */}
+              <Stack horizontal className={classNames.titleBar}>
+                <ErrorIcon />
+                <Text variant="mediumPlus" className={classNames.titleBarLabel}>Error instances</Text>
+                <Text variant="large" className={classNames.titleBarNumber}>6170</Text>
+              </Stack>
+              <Stack.Item style={{height: '100%', overflow: 'scroll'}}>
+                <ImageList imageDim={this.state.imageDim} openPanel={this.onButtonClick}/>
+              </Stack.Item>
+            </Stack>
+            <Stack style={{ width: '50%', height: '100%' }}>
+              {/* Title Bar */}
+              <Stack horizontal className={classNames.titleBar}>
+                <SuccessIcon />
+                <Text variant="mediumPlus" className={classNames.titleBarLabel}>Success instances</Text>
+                <Text variant="large" className={classNames.titleBarNumber}>3825</Text>
+              </Stack>
+              <Stack.Item style={{height: '100%', overflow: 'scroll'}}>
+                <ImageList imageDim={this.state.imageDim} openPanel={this.onButtonClick}/>
+              </Stack.Item>
+            </Stack>
+          </Stack>
+        </Stack>
+        }
+        {this.state.selectedKey == VisionDatasetExplorerTabOptions.Second && 
+        <Stack.Item>
+          <Text>{VisionDatasetExplorerTabOptions.Second}</Text>
+        </Stack.Item>
+        }
+        <Stack.Item>
+          <Flyout isOpen={this.state.panelOpen} item={{title: 'hi', image: 'hi'}} callback={this.onButtonClick}/>
```

Comment:
hmm what is this used for:
```
item={{title: 'hi', image: 'hi'}}
```
maybe remove item for now, unless it is necessary?